Add the -m option to man's command line to augment the list of paths searched by man.
The required -m option is only available in the mandoc implementation.
--- a/adapter/protocol/man.nim
+++ b/adapter/protocol/man.nim
@@ -290,7 +290,7 @@ proc myOpen(cmd: string): Opt[tuple[ofile, efile: AChaFile]] =
 proc doMan(man, keyword, section: string) =
   let sectionOpt = if section == "": "" else: ' ' & quoteShellPosix(section)
   let cmd = "MANCOLOR=1 GROFF_NO_SGR=1 MAN_KEEP_FORMATTING=1 " &
-    man & sectionOpt & ' ' & quoteShellPosix(keyword)
+    man & sectionOpt & " -m @out@ " & quoteShellPosix(keyword)
   let (ofile, efile) = myOpen(cmd).orDie(ceInternalError, "failed to run man")
   var manword = keyword
   if section != "":
